openwisp_controller/config/admin.py

```diff
@@ -814,6 +814,7 @@ class TemplateAdmin(MultitenantAdminMixin, BaseConfigAdmin, SystemDefinedVariabl
         'modified',
     ]
     readonly_fields = ['system_context']
+    autocomplete_fields = ['vpn']
```

Comment:
After this change, the non-superusers won't be able to create templates for Shared VPNs

Reply:
ok :+1: